entropylab/results_backend/sqlalchemy/db.py

```diff
@@ -1,416 +1,466 @@
-import os

```

Comment:
please fix line endings here as well

Reply:
From what I see, the file has CRLF on main and my PR changes it to LF. Hence the extra large diff. Can I ask you to confirm and resolve?